### ISSUE TITLE
Reminders are not sent

### ISSUE THREAD

**Issue by tecnomur** [invoiceninja/dockerfiles#314]
![Captura de pantalla 2021-03-31 180441](https://user-images.githubusercontent.com/47739534/113175318-9716fa00-924b-11eb-88cf-f1c29ef4f3b7.png)

Mail works fine, but i notice auto reminders doesnt.
Backups dont work either.

**Comment by turbo124** [comment]
Hi There

Can you follow the issue templates please, it makes it easier to support these issues. I need to which version you are on.

Also, what do you mean by backups dont work either?

**Comment by jinjie** [comment]
I also realised that reminders are also not sending out.

I have set up Reminder 1 and Reminder 2. 7 days and 2 days respectively, "Before the due date". There's an invoice that's due on 31 May 2021, but no reminder was sent out 7 / 2 days ago.

Not sure how am I able to troubleshoot this.

`docker exec invoiceninja_app_1 date` returns a UTC date, should I be setting the timezone correctly? Even if it is a timezone issue, I believe it would have sent out the reminders, but at a different timing but it is not.
